object/types.proto

```diff
@@ -29,6 +30,9 @@ enum ObjectType {
 
   // Object lock
   LOCK = 3;
+
+  // Initialization part of the but split object
+  INIT = 4;
```

Comment:
no payload for init objects for now. have not found requirements in it, info form the `parent_header` field is enough

Reply:
Allowing payload can make it more efficient. At the same time, likely this can be extended in future.

Reply:
> Allowing payload can make it more efficient.

Only if it has a separate message without useless (unknown for the init object) fields. But on the other hand, it breaks our rule: metabase is for header info only (but the link object has already done it too in this PR).

Reply:
> metabase is for header info only

I'm not sure what's the conflict. Header goes into meta, payload is not needed there. Link object is a different story.